### ISSUE TITLE
8.1.x: incorrect and slow display on mobile devices (iOS / Safari)

### ISSUE THREAD

**Issue by jostrasser** [grafana/grafana#37778]
**What happened**:
Since updating to 8.1.1 many panels are missing or not loading on iOS / Safari (see screen captures)
Tested with iOS 14.x and iOS 15 beta (new Safari version).

**What you expected to happen**:
All panels should be displayed correctly while scrolling.

**How to reproduce it (as minimally and precisely as possible)**:
Upgrade to 8.1.0 or 8.1.1 and view dashboards via mobile devices

**Anything else we need to know?**:
Since 8.0 the loading on mobile devices seems to work slower, lazy loading?
You can see it in the videos... it takes some time that a panel will be displayed when a user scrolls.
Is there a way to disable it?

**Environment**:
- Grafana version: 8.1.1
- Data source type & version: InfluxDB
- OS Grafana is installed on: Ubuntu bionic
- User OS & Browser: iOS 14.7 / iOS 15 beta / Safari browser

**Screen Recordings / Comparison** (GIFs are too big to post):
[Grafana 8.1.1 on iOS](https://vimeo.com/585655356)
[Grafana 8.0.6 on iOS](https://vimeo.com/585655468)

**Comment by mariusbach** [comment]
I have the same issue. I have quite a number of panels (approx. 40, most stat and about 8 graphs), so I can scroll quite some way up and down. The panels seem to only update once you scroll past them, so they are out of sight. You need to scroll away approx. 1-2 "screen contents". Once you scroll back, the panel updates only once you stop scrolling and only if it is back in sight at that moment.

In essence, it is now impossible to load the grafana page and quickly scroll through the readings. You need to a) scroll away from the panels you see when opening the browser and you need to b) stop scrolling at each panel to let it update.

Before, you'd open the browser and all panels are updating, so you were able to quickly scroll through them.

For example, selecting a different time range causes the same issue. You have to scroll back and forth, to let the chart update.

**Comment by gustavoghpf** [comment]
I noticed the same problem after upgrading to 8.1.x

**Comment by maxvonroden** [comment]
Yes, we've experienced the same problems in 8.x (currently 8.1.1). 
Sadly, this makes some dashboards unusable on mobile devices, as several panels (Graph old, Tim series, Table, Stat...) are only loading very occasionally.

Would be great, if this could be solved in 8.1.3 - x or 8.2.x.
Thanks in advance ;-)

**Comment by frepke** [comment]
Same problem here for v8.1.x

Some panels aren't loading at all (Bar gauge in my case), also not when scrolling all the way to the end and up again.
Sometime they appear when I zoom out to 50% in Safari (iOS14) and back to 85% up to 100%.
For v8.x.x: graphics are loading lazy and is not pleasant to work with. So for now I'm back to v7.5.10 which is very responsive and shows all the graphics instantly.

**Comment by hugohaggmark** [comment]
Hi everyone, if you have a chance to try out https://github.com/grafana/grafana/pull/39991 it would be appreciated. Thank you.

**Comment by gustavoghpf** [comment]
@hugohaggmark I tested version 8.2.1 but the problem persists. I noticed that the problem seems to happen in all browsers on mobile devices. Below is a test I did with Google Chrome on a Galaxy S10.
I also noticed that the problem only happens on dashboards with a large number of panels, as if it were related to the number of panels. For example, if I remove some panels from the beginning, some panels that didn't load at the end start loading.

https://user-images.githubusercontent.com/25169697/136860199-95dbc16b-b847-47bc-b630-99d9c170826e.mp4

**Comment by maxvonroden** [comment]
> Hi everyone, if you have a chance to try out #39991 it would be appreciated. Thank you.
I can also confirm that the bug isn't fixed in 8.2.1. Thanks for looking into this again.

**Comment by jostrasser** [comment]
> Problem still exists in v8.2.3 :(
> 
> Back to v7.5.10 again

@frepke 

Is 8.2.3 released already?

**Comment by frepke** [comment]
> > Problem still exists in v8.2.3 :(
> > Back to v7.5.10 again
> 
> @frepke
> 
> Is 8.2.3 released already?

Yes, otherwise I couldn't test is 😄 
[https://grafana.com/grafana/download?edition=enterprise](url)

**Comment by jostrasser** [comment]
> > > Problem still exists in v8.2.3 :(
> 
> > > Back to v7.5.10 again
> 
> > 
> 
> > @frepke
> 
> > 
> 
> > Is 8.2.3 released already?
> 
> 
> 
> Yes, otherwise I couldn't test it 😄 
> 
> [https://grafana.com/grafana/download?edition=enterprise](https://grafana.com/grafana/download?edition=enterprise)

Thanks, interesting...

Can't see it on GitHub 🥸 

![image](https://user-images.githubusercontent.com/14274103/140083316-637a81d6-f222-4051-8e46-29342c631fd5.jpeg)

**Comment by frepke** [comment]
That's weird

**Comment by gustavoghpf** [comment]
I tested version 8.2.3 and also confirm that it didn't solve the issue

**Comment by hugohaggmark** [comment]
@jostrasser and @gustavoghpf thank you for testing this out, there was a slight change in the releases because of a security fix 8.2.3 https://grafana.com/docs/grafana/latest/release-notes/release-notes-8-2-3/ doesn't include the fix for this. It should be shipped in 8.2.4. Again thanks for trying it out ⭐

**Comment by frepke** [comment]
Now, in version 8.2.4 I can confirm it's working although building the screen is very, very slow.

**Comment by jostrasser** [comment]
> Now, in version 8.2.4, I can confirm it's working although building the screen for the 1st time is very, very slow.

Hi, I can confirm that it's working also for me after upgrading to 8.2.4.
